Makefile

```diff
@@ -64,13 +64,15 @@ INCLUDEOPTS=$(shell gcc -E -x c++ - -v < /dev/null 2>&1 >/dev/null \
 	    | grep '^ /' | grep -v cc1plus | awk '{print "-I" $$1 " "}')
 PYINCLUDE=$(shell python -c \
 	  "import sysconfig;print(sysconfig.get_path('include'))")
+SYSPYINCLUDE=$(shell python -c \
+      "from __future__ import print_function; from distutils.sysconfig import get_python_inc; print(get_python_inc())")
```

Comment:
Python packaging hell 😢 

In a `conda` env both methods give the same path: the one that belongs to the `python` from the conda env. Annoying that neither of these methods gives the correct answer in both cases (inside and outside of a venv).